recipe/patches/disable-sage-location.patch

```diff
@@ -0,0 +1,12 @@
+--- src/bin/sage-location.old   2017-03-17 08:01:42.147745511 +0530
++++ src/bin/sage-location       2017-03-17 08:01:52.239787029 +0530
+@@ -299,7 +299,7 @@
+ """
+ 
+ 
+-if __name__ == '__main__':
++def main():
```

Comment:
Changed here, to disable the script. Thanks to @saraedum for finding that `sage-location` was breaking conda.